strawberry/cli/commands/export_schema.py

```diff
@@ -0,0 +1,13 @@
+import importlib
+
+import click
+
+from strawberry.printer import print_schema
+
+
+@click.command("export_schema", short_help="Exports the schema")
+@click.argument("module", type=str)
+def export_schema(module: str):
+    schema_module = importlib.import_module(module)
+    schema = getattr(schema_module, "schema")
```

Comment:
Can the `schema` symbol name be made configurabl?

Reply:
Now it can. Do we want the symbol name do be optional and default to `schema`? I made it required for now too keep it simple.

Reply:
I think `schema` is a reasonable default.

Do you think the module name and the symbol should both be separate arguments?

Reply:
I'm use to the `package.module:symbol` notation from the python unnitest and tox cli. There they allow the user to select individual tests from modules and classes within them. Thats why I used it here. I must admit though, that having a single argument is harder to validate than having two separate arguments.

Reply:
I'm not opposed to the single argument, esp. if that's what other libraries do. Although personally I'm more of a fan of pytest's double colon separator `::` and this library uses pytest. 

If we do you one, argument, it does make it harder (or less obvious) to provide a default though

Reply:
Good points. I think whatever we decide for this command should also be applied to the existing `server` command for consistency. The `server` command is currently invoke like this: `strawberry server app` where `app` is the name of a module a `schema` symbol is expected in. The strawberry server command reminds me of the way other servers are started. Uvicorn for example uses `uvicorn example:app`. For the same reason I'm not completely convinced by the double colons, but I dont mind them too much tbh.

Reply:
+1 to the server command also using the `:` notation.

Reply:
Let's use `:` as seems to be more common than's pytest's `::`, at least from what I've seen 😊

Reply:
Seems that `:` appears to be the consensus. 

side note: I think the `::` syntax pytest uses is supposed to be similar to C++'s namespace operator.

Reply:
I went with `:` and `schema` as default symbol name. I also added a little `importer` util that can be reused by the server command. In order to properly test the `importer` util I created a sample package and module inside `tests/fixtures`. Feel free to tell me to put those files somewhere else.